Src/lib/UL/spinner.c

```diff
@@ -125,11 +133,18 @@ e_spinner_err_t spinner_deinit(void)
 
 e_spinner_err_t spinner_run(void)
 {
-    static uint32_t       period                 = 400;
-    static uint32_t       ms_from_last_operation = 0;
-    static int8_t         count                  = 0;
-    static spinner_ctrl_t objL_item              = {0};
-    button_run();
+    static uint32_t       ms_from_last_operation   = 0;
```

Comment:
Looks like it is to many local variables in this function. I assume that it is possible to hide in drivers level or smth like that

Reply:
Try to keep functions as simple as possible

Reply:
Done